rpcserver.go

```diff
@@ -2890,7 +2943,11 @@ func ValidateAddress(w *Wallet, chainSvr *chain.Client, icmd btcjson.Cmd) (inter
 	switch ma := ainfo.(type) {
 	case waddrmgr.ManagedPubKeyAddress:
 		result.IsCompressed = ma.Compressed()
-		result.PubKey = ma.ExportPubKey()
+		pubKey, err := ma.ExportPubKey()
+		if err != nil {
+			return nil, ErrAddressNotInWallet
```

Comment:
I'm pretty sure this should just be a break instead of an error return.  Just because the wallet doesn't have it, does not mean it's an invalid address.  See the code below for errors on `ma.Script()`.

Also note that the call to `ma.ExportPubKey` needs to be before setting `result.IsCompressed` because that information is unknown if the pubkey isnt available.